dataset/dictionary/foods.json5

```
@@ -142,175 +142,205 @@
     ja: "北地のスモークチキン",
+    zhCN: "北地烟熏鸡",
     pronunciationJa: "ほくちのスモークチキン",
     tags: [ "food" ],
   },
   {
     en: "Cream Stew",
     ja: "ホワイトソースポトフ",
+    zhCN: "白汁时蔬烩肉", // Note: zhTW: "白醬時蔬燴肉"
     tags: [ "food" ],
   },
   {
     en: "Tea Break Pancake",
     ja: "午後のパンケーキ",
+    zhCN: "庄园烤松饼",
     pronunciationJa: "ごごのパンケーキ",
     tags: [ "food" ],
   },
   {
     en: "Fried Radish Balls",
     ja: "大根の揚げ団子",
+    zhCN: "炸萝卜丸子",
     pronunciationJa: "だいこんのあげだんご",
     tags: [ "food" ],
   },
   {
     en: "Fisherman's Toast",
     ja: "漁師トースト",
+    zhCN: "渔人吐司",
     pronunciationJa: "りょうしトースト",
     tags: [ "food" ],
   },
   {
     en: "Satisfying Salad",
     ja: "満足サラダ",
+    zhCN: "满足沙拉",
     pronunciationJa: "まんぞくサラダ",
     tags: [ "food" ],
   },
   {
     en: "Goulash",
     ja: "お肉と野菜のシチュー",
+    zhCN: "稠汁蔬菜炖肉",
     pronunciationJa: "おにくとやさいのシチュー",
     tags: [ "food" ],
   },
   {
     en: "Almond Tofu",
     ja: "杏仁豆腐",
+    zhCN: "杏仁豆腐",
     pronunciationJa: "あんにんどうふ",
     tags: [ "food" ],
   },
   {
     en: "Jueyun Chili Chicken",
     ja: "椒椒鶏",
+    zhCN: "香嫩椒椒鸡",
     pronunciationJa: "ジョジョジー",
     tags: [ "food" ],
   },
   {
     en: "Jewelry Soup",
     ja: "真珠翡翠白玉湯",
+    zhCN: "珍珠翡翠白玉汤",
     pronunciationJa: "しんじゅひすいはくぎょくとう",
     tags: [ "food" ],
   },
   {
     en: "Matsutake Meat Rolls",
     ja: "松茸の肉巻き",
+    zhCN: "松茸酿肉卷",
     pronunciationJa: "まつたけのにくまき",
     tags: [ "food" ],
   },
   {
     en: "Crystal Shrimp",
     ja: "水晶蝦",
+    zhCN: "水晶虾",
     pronunciationJa: "すいしょうえび",
     tags: [ "food" ],
   },
   {
     en: "Lotus Seed and Bird Egg Soup",
     ja: "ハスの実入り茶碗蒸し",
+    zhCN: "莲子禽蛋羹",
     pronunciationJa: "ハスのみいりちゃわんむし",
     tags: [ "food" ],
   },
   {
     en: "Noodles with Mountain Delicacies",
     ja: "山幸の麺",
+    zhCN: "山珍热卤面",
     pronunciationJa: "やまさちのめん",
     tags: [ "food" ],
   },
   {
     en: "Mint Salad",
     ja: "ミントの和え物",
+    zhCN: "凉拌薄荷",
     pronunciationJa: "ミントのあえもの",
     tags: [ "food" ],
   },
   {
     en: "Crab Roe Tofu",
     ja: "かにみそ豆腐",
+    zhCN: "蟹黄豆腐",
     pronunciationJa: "かにみそどうふ",
     tags: [ "food" ],
   },
   {
     en: "Tuna Sushi",
     ja: "マグロの握り寿司",
+    zhCN: "金枪鱼寿司",
     pronunciationJa: "マグロのにぎりずし",
     tags: [ "food" ],
   },
   {
     en: "Onigiri",
     ja: "おにぎり",
+    zhCN: "饭团",
     tags: [ "food" ],
   },
   {
     en: "Mixed Yakisoba",
     ja: "焼きそば",
+    zhCN: "什锦炒面",
     pronunciationJa: "やきそば",
     tags: [ "food" ],
   },
   {
     en: "Crab Roe Kourayaki",
     ja: "かにみそ甲羅焼き",
+    zhCN: "蟹黄壳壳烧",
     pronunciationJa: "かにみそこうらやき",
     tags: [ "food" ],
   },
   {
     en: "Wakatakeni",
     ja: "若竹煮",
+    zhCN: "若竹煮",
     pronunciationJa: "わかたけに",
     tags: [ "food" ],
   },
   {
     en: "Radish and Fish Stew",
     ja: "魚とダイコンの煮込み",
+    zhCN: "鲜鱼炖萝卜",
     pronunciationJa: "さかなとダイコンのにこみ",
     tags: [ "food" ],
   },
   {
     en: "Berry Mizu Manjuu",
     ja: "ラスベリー水まんじゅう",
+    zhCN: "树莓水馒头,
```

You need a doublequote here and I think this is the cause of the error.
```suggestion
    zhCN: "树莓水馒头",
```

---

Oh thank you!